packages/donations-app/app/index.html

```diff
@@ -27,22 +27,44 @@
   <!-- Hotjar Tracking Code for https://stage.alice.si/ -->
   <script>
     (function(h,o,t,j,a,r){
-        // Added by alex@alice.si to disable hotjar on production
+        var hotjarLoadingStarted = false;
+
         h.hj=h.hj||function(){(h.hj.q=h.hj.q||[]).push(arguments)};
         if (location.href.indexOf('donationsapp.alice.si') != -1) {
+          hotjarLoadingStarted = true;
           console.log('Hotjar loading started on production');
           h._hjSettings={hjid:1481120,hjsv:6};
         } else if (location.href.indexOf('stage.alice.si') != -1) {
+          hotjarLoadingStarted = true;
           console.log('Hotjar loading started on stage');
           h._hjSettings={hjid:1451891,hjsv:6};
         }
-        a=o.getElementsByTagName('head')[0];
-        r=o.createElement('script');r.async=1;
-        r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
-        a.appendChild(r);
+
+        if (hotjarLoadingStarted) {
+          a=o.getElementsByTagName('head')[0];
+          r=o.createElement('script');r.async=1;
+          r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
+          a.appendChild(r);
+        }
     })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
   </script>
 
+  <script>
+    (function () {
+      var timer = setInterval(function () {
```

Comment:
What do you think about Mutation Observe (https://stackoverflow.com/questions/3219758/detect-changes-in-the-dom), is it worth giving a try?

Reply:
I have added a task on JIRA to check it later